README.md

```diff
@@ -185,20 +185,21 @@ Nano ID could be started from number. HTML ID can’t be started from the numbe
 
 ### React Native
 
-To generate secure random IDs in React Native, you must use
-[a native random generator] and asynchronous API:
+To generate secure random IDs in React Native, you must use a native random generator (like [expo-random] or [react-native-securerandom]) and the asynchronous API:
 
-```js
-const generateSecureRandom = require('react-native-securerandom').generateSecureRandom
-const format = require('nanoid/async/format')
-const url = require('nanoid/url')
+```ts
+import { getRandomBytesAsync } from 'expo-random';
+import format from 'nanoid/async/format';
+import url from 'nanoid/url';
 
-async function createUser () {
-  user.id = await format(generateSecureRandom, url, 21);
-}
+// Only required if you use TypeScript to tell the compiler that the types are compatible.
+const random = (getRandomBytesAsync as unknown) as (byteCount: number) => Promise<number[]>;
```

Comment:
I use pure JS everywhere. Let’s use pure JS here too. TS users will find a way to do it.